ArtisansFinal/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
+        classpath 'com.android.tools.build:gradle:3.3.1'
```

Comment:
please pull from upstream before submitting a pr.. the current version is ```3.2.1``` to help use a library and so let's keep it at that